.github/workflows/release.yml

```
@@ -0,0 +1,137 @@
+# This action is centrally managed in https://github.com/asyncapi/.github/
+# Don't make changes to this file in this repo as they will be overwritten with changes made to the same file in the above-mentioned repo
+
+# It does magic only if there is a package.json file in the root of the project
+name: Release - if Node project
```

```suggestion
name: Release with changeset
```

can you please name file like `release-with-changeset.yml` as might be that this release will work also in our generator and then also other monorepos, so we will maintain it in `.github` and we will do it with `release-with-changeset.yml` name - so once you opt in in future for file from `.github`, it will just be overwritten

---

ok sure changed the name as required.

---

ok changed to the changesets.